test/sqllogictest/string.slt

```diff
@@ -939,6 +939,128 @@ SELECT position(e'\u0903\u0951' IN e'\u0930\u0903\u0951')
 ----
 2
 
+### left ###
+statement ok
+CREATE TABLE lefttest (strcol char(15), vccol varchar(15), smicol smallint, intcol int)
+
+statement ok
+INSERT INTO lefttest VALUES ('Mg', 'Mn', 1, -1), ('magnesium', 'manganese', 3, NULL),
+    (NULL, NULL, 0, 0), ('24.31', '54.94', 3, -3), ('长久不见', '爱不释手', NULL, 3),
+    ('', '', -1, 2)
+
+# invalid input
+statement error
+SELECT left(42)
+
+statement error
+SELECT left('str')
+
+statement error
+SELECT left(42, 'str')
+
+statement error
+SELECT left('str', 42, 17)
+
+# standard tests
+
+#TODO: materialize#589 select left(strcol, foo) from lefttest
+
+# edge case
+query T rowsort
+SELECT left(vccol, 0) FROM lefttest
+----
+(empty)
+(empty)
+NULL
+(empty)
+(empty)
+(empty)
+
+# from the back
```

Comment:
Shouldn't this be "from the front"? And later you have "from the back", which should be exchanged.

Reply:
Good catch, my search&replace didn't match it :D